### PR TITLE
[BPK-1635] Web navigation bar should accept title as node

### PR DESCRIPTION
**Pull request by matthewdavidson**
Checked in Chrome, IE11, Safari, FF, Edge, Mobile Safari & Chrome on Android.

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -11,20 +11,16 @@
 # 2018-06-20 - Add gradient definitions to React Native
 
 **Breaking:**
-
```
Comment (matthewdavidson) [inline changelog.md]: Staying consistent with the rest of the changelog by removing newlines after "titles"...

File: packages/bpk-component-navigation-bar/src/BpkNavigationBar-test.js [hunk 2]
```diff
@@ -71,6 +93,7 @@ describe('BpkNavigationBar', () => {
         />,
       )
       .toJSON();
+
```
Comment (shaundon) [inline packages/bpk-component-navigation-bar/src/BpkNavigationBar-test.js]: I can just hear you saying "let the code breathe" in my head

File: packages/bpk-component-navigation-bar/src/BpkNavigationBar-test.js [hunk 3]
```diff
@@ -27,8 +27,29 @@ import BpkNavigationIconButton from './BpkNavigationBarIconButton';
 describe('BpkNavigationBar', () => {
   it('should render correctly', () => {
     const tree = renderer
-      .create(<BpkNavigationBar id="test" title="test" aria-label="test" />)
+      .create(<BpkNavigationBar id="test" title="test" />)
```
Comment (matthewdavidson) [inline packages/bpk-component-navigation-bar/src/BpkNavigationBar-test.js]: Created a separate test case for arbitrary props...

File: packages/bpk-component-navigation-bar/src/BpkNavigationBar.js [hunk 4]
```diff
@@ -50,29 +50,35 @@ const BpkNavigationBar = (props: Props) => {
...
+        <BpkText
+          id={titleId}
+          bold
+          className={getClassNames('bpk-navigation-bar__title')}
+        >
+          {title}
+        </BpkText>
+      ) : (
+        title
+      )}
```
Comment (matthewdavidson) [inline packages/bpk-component-navigation-bar/src/BpkNavigationBar.js]: If consumers pass anything other than a string we do not wrap in `BpkText`...

File: changelog.md [hunk 5]
```diff
@@ -11,20 +11,16 @@
...
 - react-native-bpk-styles: 1.0.9 => 1.1.0
   - Added defintitions for the Skyscanner primary gradient. `react-native-bpk-styles` now has a peer depedency on `react-native-linear-gradient`.
 
 **Fixed:**
-
 - bpk-component-navigation-bar: 1.2.3 => 1.2.4
-  - `title` now accepts strings, in addition to nodes
-
+  - `title` now accepts strings, in addition to elements
```
Comment (shaundon) [inline changelog.md]: Sentence should end with a full stop 🙈

Comment (matthewdavidson) [reply changelog.md]: appeased ... gimme another ✅ please :)

**Comment by matthewdavidson** [comment]
Oops, I forgot to squash 🙄

**Pull request merged**